libs/community/langchain_google_community/docai.py

```diff
@@ -306,6 +381,9 @@ def docai_parse(
             field_mask: a comma-separated list of which fields to include in the
                 Document AI response.
                 suggested: "text,pages.pageNumber,pages.layout"
+            chunk_size: the maximum number of characters per chunk.
+            include_ancestor_headings: whether or not to include ancestor
+                headings when splitting.
```

Comment:
Add note that this is only supported in layout parser.

I wonder if this could be refactored to reduce duplication. These parameters are added in 3 different methods.

Reply:
agree. I was trying to minimize changes to existing implementation. One option is to create a helper function to prepare process options and in the calling functions such as `online_process`, `batch_parse` and 'docai_parse` users can pass processing options as kwargs.